modules/linear_boltzmann_solvers/a_lbs_solver/acceleration/diffusion_mip_solver.cc

```diff
@@ -594,8 +594,8 @@ DiffusionMIPSolver::AssembleAand_b(const std::vector<double>& q_vector)
     const auto cc_nodes = cell_mapping.GetNodeLocations();
     const auto& unit_cell_matrices = unit_cell_matrices_[cell.local_id_];
 
-    const auto& cell_K_matrix = unit_cell_matrices.K_matrix;
-    const auto& cell_M_matrix = unit_cell_matrices.M_matrix;
+    const auto& cell_K_matrix = unit_cell_matrices.intV_gradshapeI_gradshapeJ;
+    const auto& cell_M_matrix = unit_cell_matrices.intV_shapeI_shapeJ;
```

Comment:
If you changed the name in `unit_cell_matrices`, should you also rename the variable, i.e. the `const auto &` one?

Reply:
Good catch. I used did a bulk rename in CLion but did not check for local variables. This has now been changed.